orchestrator/api/api_v1/endpoints/settings.py

```diff
@@ -86,12 +85,35 @@ async def set_global_status(
     if websocket_manager.enabled:
         # send engine status to socket.
         await websocket_manager.broadcast_data(
-            [WS_CHANNELS.ENGINE_SETTINGS], {"engine-status": generate_engine_status_response(result)}
+            [WS_CHANNELS.ENGINE_SETTINGS],
+            {"engine-status": generate_engine_status_response(result)},
         )
 
     return status_response
 
 
+def get_worker_status() -> WorkerStatus:
+    """
+    Return information job workers and queues.
+
+    Returns:
+    - The number of queued jobs
+    - The number of workers
+    - The number of running jobs
+    - The number of successful and unsuccessful jobs
+    """
+
+    response: WorkerStatus
```

Comment:
Is this needed since both classes are derived from WorkerStatus? Or maybe just a return from within the if/else instead of a variable.

Reply:
mypy was complaining. Fixed it